packages/plugin-copying-task/examples/example.html

```diff
@@ -1,16 +1,17 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <script src="https://unpkg.com/jspsych@7.3.0"></script>
-    <script src="https://unpkg.com/@jspsych/plugin-preload@1.0.0"></script>
-    <link rel="stylesheet" href="https://unpkg.com/jspsych@7.0.0/css/jspsych.css">
-    
-    <script src="https://www.unpkg.com/fabric@5.2.4-browser/dist/fabric.js"></script>
-    
-    <script src="../src/index.js"></script>
-  
-</head>
-  <body></body>
+    <head>
+        <script src="https://unpkg.com/jspsych@8.0.0"></script>
+        <script src="https://unpkg.com/@jspsych/plugin-preload@2.0.0"></script>
+        <link rel="stylesheet" href="https://unpkg.com/jspsych@8.0.0/css/jspsych.css">
+        
+        <script src="https://www.unpkg.com/fabric@5.2.4-browser/dist/fabric.js"></script>
+        
+        <script src="../dist/index.browser.min.js"></script>
```

Comment:
maybe not the minified version as the default, to allow for easier console debugging?